### DISCUSSION TITLE
Dashboard link are not shown after update

### DISCUSSION THREAD

**Discussion by 011248163264** [Security-Onion-Solutions/securityonion#8046] [Unsupported Versions] [answered]
Hi i have maked a update from 2.3.90 to 2.3.120. the update is successful without errors.
But the dashboard link in the menu is not shown on the mainpage.

What can i do?

**Comment by xfaith** [comment, marked as answer]
Which Dashboard?  
If you are talking about the Dashboard link from the Left Pane on the SOC Main page will be available in 2.3.130 per SecurityOnion blog

**Reply by 011248163264** [reply]
Aho oh ja haha ok sorry :-)
